queries.sql

```diff
@@ -0,0 +1,69 @@
+DROP TABLE IF EXISTS organization;
+DROP TABLE IF EXISTS channels;
+DROP TABLE IF EXISTS messages;
+
+CREATE TABLE organization (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    organization_name VARCHAR(128),
```

Comment:
Because of the `,` at the end of last statement it will throw syntax errors. It applies to all other tables.